nautobot/extras/jobs.py

```diff
@@ -343,7 +347,12 @@ def deserialize_data(cls, data):
         return_data = {}
 
         for field_name, value in data.items():
-            var = vars[field_name]
+            # If a field isn't a var, skip it (e.g. `_commit`).
+            try:
+                var = vars[field_name]
+            except KeyError:
+                continue
+
```

Comment:
For backwards compatibility, shouldn't we be deserializing string values back into netaddr objects?

Reply:
I wasn't sure, because from what I could tell, what comes thru as `data` is still coming from the form? As you see in the new test job's `run()` method, I'm explicitly casting the field values to strings because they were still instances. I fully admit I may be missing something.

Reply:
Testing with 1.0.3 shows that they are netaddr objects:

![image](https://user-images.githubusercontent.com/5603551/128254895-08d6c380-ca96-4c67-9c7d-ab50c6b88df5.png)

With your branch they are currently strings:

![image](https://user-images.githubusercontent.com/5603551/128255218-6acb0aa4-c1b9-4c07-aa7d-e4dc0d957770.png)

Reply:
I see now that the way we are testing is not actually valid because it's not running the job thru a worker. Grunt.